web3.js/test/vote-program.test.ts

```diff
@@ -154,13 +173,57 @@ describe('VoteProgram', () => {
         lamports: LAMPORTS_PER_SOL,
         toPubkey: recipient.publicKey,
       });
-
       await sendAndConfirmTransaction(connection, withdraw, [authorized], {
         preflightCommitment: 'confirmed',
       });
       expect(await connection.getBalance(recipient.publicKey)).to.eq(
         LAMPORTS_PER_SOL,
       );
+
+      const newAuthorized = Keypair.generate();
```

Comment:
Can you also add a case to test that the authorized withdrawer can sign to authorize a new Voter? More of a demo to the user than testing anything in your implementation.
ref: https://github.com/solana-labs/solana/pull/22522
Might need to set up separate newVoter and newWithdrawer Keypairs here.